wizard.sh

```diff
@@ -90,7 +89,12 @@ detect_services () {
     for SVC in ${SUPPORTED_SERVICES} ; do
         log_info "Checking if service '${SVC}' is running (ps+systemd)"
         for SRC in "${SYSTEMD_SERVICES}" "${PSAX}" ; do
-            echo ${SRC} | grep ${SVC} >/dev/null
+            if [$SRC != "apache2" ] ; then 
```

Comment:
I would suggest adding `httpd` to the `SUPPORTED_SERVICES` and substitute it with `apache2` just before we add it to the `DETECTED_SERVICES` array :


in the `if [ $? -eq 0 ];` block before adding `SVC` to the detected service array :

```bash
#on centos, apache2 is named httpd                                                                                                                                                                                            
if [[ ${SVC} == "httpd" ]] ; then
   SVC="apache2";
fi
```


I think it will make the code easier to read.

Reply:
That's more concise 

Done :)